tools/src/block-gen/Main.hs

```diff
@@ -51,14 +52,17 @@ main = flip catch catchEx $ do
 
     let nodes = length invSecretsMap
     let flatDistr = FlatStakes (fromIntegral nodes) (mkCoin $ fromIntegral nodes)
-    let bootStakeholders =
-            GenesisWStakeholders $ HM.fromList $
-            zip (HM.keys $ unInvSecretsMap invSecretsMap) (repeat 1)
+    let bootStakeholders
+            | isDevelopment =
+                GenesisWStakeholders $ HM.fromList $
```

Comment:
I and @georgeee decided not to add other distribution and just created tasks for it. we hoped it will be added when it's needed